packages/legacy/core/App/components/misc/CredentialCard11.tsx

```diff
@@ -429,7 +455,12 @@ const CredentialCard11: React.FC<CredentialCard11Props> = ({
     )
   }
   return overlay.bundle ? (
-    <View style={[styles.container, style, { elevation: elevated ? 5 : 0 }]}>
+    <View
+      onLayout={(event) => {
```

Comment:
What's delay like before this `onLayout` takes effect?

Reply:
the delay is maybe a quarter second

Reply:
So there's a quarter second with no watermark? I wonder if there's another way to do it...